### ISSUE TITLE
Did not work with atom 1.9.0-beta0

### ISSUE THREAD

**Issue by t246246** [Glavin001/atom-beautify#1017]
### Description
Did not work at all with atom 1.9.0-beta0. I tried on Windows 7 64bit.
Nothing happened with the following error for JavaScript/JSON/HTML files.

```
TypeError: Cannot read property 'getView' of undefined
    at getScrollTop (file:///C:/Users/t246246/.atom/packages/atom-beautify/src/beautify.coffee:31:29)
    at beautifyCompleted (file:///C:/Users/t246246/.atom/packages/atom-beautify/src/beautify.coffee:108:27)
    at tryCatcher (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:502:31)
    at Promise._settlePromise (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:559:18)
    at Promise._settlePromise0 (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:604:10)
    at Promise._settlePromises (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:683:18)
    at Async._drainQueue (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:138:16)
    at Async._drainQueues (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:148:10)
    at Async.drainQueues (C:\Users\t246246\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:17:14)
```

**Comment by jonyeezs** [comment]
I get the same problem as well. Here's my log file as requested: https://gist.github.com/jonyeezs/467ee491b57c38981c7c3e7f94f28ec5

@t246246 i recommend you do the same as well. Right-click on the troubled file and select 'Debug Atom Beautify'

**Comment by Glavin001** [comment]
This is a duplicate of #979 and fixed with #987 which has been merged and not yet published, since I was waiting for other Pull Requests and 1.9 is not released as stable yet. I will publish it now.

**Comment by Glavin001** [comment]
Published to v0.29.8
